src/read.rs

```diff
@@ -645,6 +645,32 @@ impl<R: Read + Seek> ZipArchive<R> {
         Ok(shared)
     }
 
+    /// Returns key and salt
+    pub fn get_aes_key_and_salt(
```

Comment:
```suggestion
    #[cfg(feature = "aes-crypto")]
    pub fn get_aes_key_and_salt(
```
This is probably why the tests are failing with `--no-default-features`.